### DISCUSSION TITLE
Success page on checkout

### DISCUSSION THREAD

**Discussion by smsalmanshahAN** [an-nadaaa/nda-front#109] [Feedback]
![image](https://user-images.githubusercontent.com/101787508/159403073-17dc2f93-c80c-4fb7-8815-0be1c0518f2a.png)

I am assuming that this will be developed later.

Things to note that message needs to include 
Thanks
how to contact for more info or invoice.
What to do next? Goto campaign page or share our project on social media. Needs more thoughts.

**Comment by an-nadaa** [comment]
This is done by now. thank you for reporting it
